styles/styles-book-detail.js

```diff
@@ -0,0 +1,192 @@
+import {css} from "lit-element";
+
+export default css`
+
+@import 'styles';
+
+.main{
+    display: flex;
+    height: 100vh;
+    align-items: center;
+    justify-content: center;
+}
+
+.header{
+    padding-bottom: 12px;
+    display: inline;
+    border-bottom: solid 4px #BED23C;
+    width: 75vh;
+}
+
+.cuerpo{
+    padding-top: 36px;
+}
+.flexcontainer{
+    display: flex;
+    width: 62vw;
+    height: 416px;
+    padding: 24px;
+    box-shadow: 5px 5px 10px rgb(0 0 0 / 50%);
+    position: absolute;
+}
+
+.content{
+    display: flex;
+    flex-direction: column;
+    padding: 0px 52px;
+}
+
+.title{
+    font-size: 32px;
+    font-weight: 500;
+    display: inline;
+}
+
+.gender{
+    color: #828282;
+    font-size: 24px;
+    font-weight: 500;
+    display: inline;
+    padding-left:10px;
+}
+
+.subtitle{
+    font-size: 20px;
+    padding-bottom: 30px;
+    display: block;
+    font-weight: 500;
+}
+
+.features{
+    color: #828282;
+    font-size: 20px;
+    line-height: 24px;
+    font-weight: 300;
+  }
+
+  .portada .cover{
+    width: 261px;
+    height: 368px;
+  }
+  
```

Comment:
Don't mix English and Spanish, fix all styles 😬

Reply:
language fixes successful